requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy==1.19.1
+opencv-python==4.3.0.36
```

Comment:
@guinnesschen Please include `requirements.txt` next time as this improves code portability.